### DISCUSSION TITLE
Any way to sort/filter forks?

### DISCUSSION THREAD

**Discussion by AlexanderDudek** [community/community#23676] [New to GitHub]
<p>Sometimes I come across an old repo and want to quickly see if anyone forked it and expanded on it, but I see no way to sort or filter forks. Would be easier to see in the overview whether they diverge or not.</p>

**Comment by canuckjacq** [comment, marked as answer]
<p>We don’t seem to have anything clever for that, but this looks promising:</p>
<p><a href="https://github.com/techgaun/active-forks">https://github.com/techgaun/active-forks</a></p>
